generator/transit_generator.cpp

```diff
@@ -213,12 +194,13 @@ void DeserializerFromJson::operator()(FeatureIdentifiers & id, char const * name
         ("Cann't convert osm id string:", osmIdStr, "to a number."));
   osm::Id const osmId(osmIdNum);
   auto const it = m_osmIdToFeatureIds.find(osmId);
-  CHECK(it != m_osmIdToFeatureIds.cend(), ("Osm id:", osmId, "(encoded", osmId.EncodedId(),
-                                           ") is not found in osm id to feature ids mapping."));
-  CHECK_EQUAL(it->second.size(), 1,
-              ("Osm id:", osmId, "(encoded", osmId.EncodedId(),
-               ") from transit graph doesn't present by a single feature in mwm."));
-  id.SetFeatureId(it->second[0]);
+  if (it != m_osmIdToFeatureIds.cend())
+  {
+    CHECK_EQUAL(it->second.size(), 1,
+                ("Osm id:", osmId, "(encoded", osmId.EncodedId(),
+                 ") from transit graph doesn't present by a single feature in mwm."));
```

Comment:
s/doesn't present/isn't represented/

Reply:
Though, I don't understand what you mean by this.
Should it be represented by exactly one feature? If yes, don't we need two separate messages for the
```< 1``` and ```> 1``` cases?
Or,
```... must be represented by exactly one feature in the mwm.```

Reply:
As you know some osm features (osm id) may be converted to several our features (feature ids). For example: http://www.openstreetmap.org/way/42798631#map=18/56.29197/43.86209

Every osm feature from transit graph json should be converted to only one our feature. But in case of features near mwm border it's possible that there's no feature id for osm id.

In short, every osm id should be converted to 0 or 1 feature id.

Reply:
Then,
```is not represented by any feature in the mwm.```